test/unit/text/shaping.cpp

```diff
@@ -65,39 +65,39 @@ TEST_CASE("shaping")
     mapnik::face_manager fm(fl, font_file_mapping, font_memory_cache);
 
     {
-        std::vector<std::tuple<unsigned, unsigned>> expected =
-            {{{0, 0}, {0, 3}, {0, 4}, {0, 7}, {3, 8}, {11, 9}, {68, 10}, {69, 11}, {70, 12}, {12, 13}}};
+        std::vector<std::pair<unsigned, unsigned>> expected =
+            {{0, 0}, {0, 3}, {0, 4}, {0, 7}, {3, 8}, {11, 9}, {68, 10}, {69, 11}, {70, 12}, {12, 13}};
         // with default NotoSans-Regular.ttc and NotoNaskhArabic-Regular.ttf ^^^
```

Comment:
Oh, gcc 5.4 :no_entry: I stopped testing with gcc 5 after recent reports that 5.3 can't compile x3 grammars, so I deemed the gcc-5 series insufficient.